node/services/loggerService.ts

```diff
@@ -7,6 +7,7 @@ import type { ServiceError } from '../typings/serviceError'
 export const loggerService = (logger: Logger): AppLogger => {
   const logError = (error: ServiceError | ResolverError) => {
     logger.error({ ...error })
+    console.error(error)
```

Comment:
I think you could restrict calling the console only if the app is linked:
```suggestion
if (process.env.VTEX_APP_LINK) console.error(error)
```

Reply:
Good idea, Conrado! Thanks, [done](https://github.com/vtex-apps/tiktok-tbp/pull/39/commits/765ea523f005e84c43a61e2565cedd42b69366dc)!